modular_splurt/code/datums/components/dullahan.dm

```diff
@@ -72,3 +99,58 @@
 	var/mob/living/carbon/human/H = quirk_holder
 	if(H.dna.features["neckfire"] && !istype(H, /mob/living/carbon/human/dummy))
 		H.AddComponent(/datum/component/neckfire, H.dna.features["neckfire_color"])
+
+/datum/component/dullahan
+	var/obj/item/clothing/head_accessory
+	var/mutable_appearance/head_accessory_MA
+
+/datum/component/dullahan/proc/add_head_accessory(obj/item/clothing/I)
+	head_accessory_MA = mutable_appearance('modular_splurt/icons/mob/clothing/current_head_accessories.dmi')
```

Comment:
Isn't it possible to just make the items use their own icons and icon states as if they were normal hats on the head?

Reply:
sure, i just didn't wanna risk someone using a weird hat or whatever

Reply:
towel, paper hat